user_guide/src/notebooks/introduction_polars.ipynb

```diff
@@ -24,7 +24,7 @@
     "* Lazily evaluated\n",
     "    - Can be optimized\n",
     "    - Gives the library writer context and informed decision can be made\n",
-    "* Embarassingly parallel\n",
+    "* Incredibly parallel\n",
```

Comment:
"embarassangly parallel" is a technical term. It must remain in this sentence. It has more guarantees than "incredibly" which is has semantic meaning.